.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
+.idea
```

Comment:
para estas cosillas hay que hacerse un [.gitignore general](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) heheh

Reply:
Feel free to ignore this comment

Reply:
It's actually a general .gitignore xD

Reply:
We can add it in further PRs  I guess